src/data/CoreData.tsx

```diff
@@ -1,5 +1,7 @@
 import { useState } from 'react';
 
+import { loadVariantTagData } from '../data/IANAData'; // issue 6
```

Comment:
Don't add PR comments. Comments should help people understand the code better. Labelling this "issue 6" doesn't help anyone understand this code.
```suggestion
import { loadVariantTagData } from '../data/IANAData';
```